cmd/podman/podman.go

```diff
@@ -0,0 +1,16 @@
+// +build linux
```

Comment:
Can we get a better name on this file? registries_linux.go and registries_unsupported.go maybe?

Reply:
i needed a generic file for stuff like this.  there will be more which is why i chose it.  how about platform.go ?

Reply:
Platform.go SGTM

Reply:
done and done